src/main/java/org/jenkinsci/plugins/workflow/cps/actions/ArgumentsActionImpl.java

```diff
@@ -276,8 +274,6 @@ Object sanitizeObjectAndRecordMutation(@CheckForNull Object o, @CheckForNull Env
         }
     }
 
-    static final OutputStream NULL_STRM = new NullOutputStream();
```

Comment:
Why did you remove this?

Reply:
Because it is a flyweight object (no fields), so there is no need to cache it.